app/controllers/publishers_controller.rb

```diff
@@ -76,8 +76,7 @@ def create
         redirect_to(root_path, notice: I18n.t("publishers.invalid_email_value") )
       end
     else
-      Rails.logger.error(I18n.t("recaptcha.errors.verification_failed"))
-      redirect_to(root_path, notice: I18n.t("recaptcha.errors.verification_failed"))
+      redirect_to root_path(:captcha => params[:captcha])
```

Comment:
could you update this to the newer colon hash syntax: `captcha: params[:captcha]`